.travis.yml

```diff
@@ -201,6 +201,7 @@ jobs:
 before_deploy:
   - npm install
   - npm run playground:build -- --public-url '/gutenberg'
+  - npm run design-system:build
```

Comment:
This will run on for all jobs when merging into `master`. We already have this issue for the playground. It will get only worse:

![Screen Shot 2019-10-04 at 07 38 25](https://user-images.githubusercontent.com/699132/66183707-0e30f980-e67a-11e9-9b44-c6d3466e46af.png)

We will need to improve the setup of Travis to run deploys only once per PR merged.

Reply:
We need two stages. I'll fix it.